src/base64.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+ *  Copyright (C) 2004-2017 Savoir-faire Linux Inc.
```

Comment:
This unit comes from https://stackoverflow.com/a/180949 and is copyrighted by René Nyffenegger. You can't relicense code like that.

Reply:
woops, sorry. Big mistake.